.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: release
```

Comment:
Please capitalize and add a trailing newline for consistency.

```suggestion
name: Release

```

Reply:
Done!